.travis.yml

```diff
@@ -1,30 +1,38 @@
 language: csharp
 
+sudo: required
+dist: trusty
+
+mono:
+  - 4.4.2
```

Comment:
Can you add "latest" and even "nightly" here too?  Perhaps drop one or two of the others